Multi-tenancy fixes (#179)
dgraph-io/dgraph-docs: content/enterprise-features/multitenancy.md

{{% notice "note" %}}
Only super-admins ([Guardians of the galaxy](#guardians-of-the-galaxy)) have access across tenants.

> The super admin is used only for database admininstration operations, such as exporting data of all tenants. Guardian of the galaxy cannot read across tenats.

Reviewer: typo `tenants`

Author: fixed

> The super admin is used only for database admininstration operations, such as exporting data of all tenants. Guardian of the galaxy users cannot read across tenants.

Reviewer 2: ```suggestion
The super admin is used only for database administration operations, such as exporting data of all tenants. _Guardian of the Galaxy_ members cannot read across tenants.
```

- [Guardians of the Galaxy](#guardians-of-the-galaxy) (Super Admins) 
- Guardians of the Namespace

>   - They can add users to groups inside the namespace

Reviewer: Why are we removing this? Guardian of a namespace can create users within their namespace.

Author: @NamanJain8 AFAIK they can only create a namespace that will also create the guardian of namespace group that has the groot user. IDK if the guardian of galaxy users can create other groups or users in other namespaces

Reviewer 3: That's right. They cannot create users for other namespaces but within the galaxy namespace, they can do anything that a guardian group can do. It includes creating users.

Author: ok then let me rephrase it better to clarify that they can only create users/groups within their namespace

- [Guardians of the Galaxy](#guardians-of-the-galaxy) (Super Admins) 
- Guardians of the Namespace

>   - They can create/add users/groups inside their own namespace

Reviewer 2: ```suggestion
  - They can create users and groups inside their own namespace
```

  - They can assign users to groups inside their own namespace
  - They can assign predicates to groups inside their own namespace
  - They can add users to groups inside the namespace
  - They can export their namespace 

>   - They can query/mutate in their namespace

Reviewer 2: ```suggestion
  - They can query and mutate in their namespace
```

<0x01> "name" "ibrahim" <0x0> .      -> this belongs to namespace 0x00
```

For example, when the _Guardian of the Galaxy_ user is used to export the namespace `0x1234` to a folder in the export directory (by default this directory is `export`):

```graphql
mutation {
  export(input: {format: "rdf", namespace: 1234}) {
    response {
      message
    }
  }
}
```

> When using the _Guardian of the Namespace_, then no need to specify the namespace in the GraphQL mutation, as they can only export within their own namespace:

Reviewer 2: ```suggestion
When using the _Guardian of the Namespace_, there's no need to specify the namespace in the GraphQL mutation, as they can only export within their own namespace:
```
